src/Lykke.Job.LiteCoin/PeriodicalHandlers/PeriodicalHandlerHost.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Common;
+using Common.Log;
+using Lykke.Job.LiteCoin.Functions;
+using Lykke.Job.LiteCoin.Settings;
+
+namespace Lykke.Job.LiteCoin.PeriodicalHandlers
+{
+    public class PeriodicalHandlerHost
```

Comment:
Since TimerTrigger is IDisposable, PeriodicalHandlerHost should be IDisposable too and should dispose all timer triggers